crates/re_sdk/src/msg_sender.rs

```diff
@@ -250,6 +330,19 @@ impl MsgSender {
         Ok(())
     }
 
+    /// Turns the current message into a single [`re_log_types::LogMsg`]
+    pub fn into_log_msg(
+        self,
+        recording_id: RecordingId,
+    ) -> Result<re_log_types::LogMsg, DataTableError> {
+        let data_table = re_log_types::DataTable::from_rows(
+            re_log_types::TableId::random(),
+            self.into_rows().into_iter().flatten(),
+        );
+        let arrow_msg = data_table.to_arrow_msg()?;
+        Ok(re_log_types::LogMsg::ArrowMsg(recording_id, arrow_msg))
+    }
```

Comment:
Don't need the SDK for this

Reply:
Can you elaborate? Is this the same complaint as in https://github.com/rerun-io/rerun/pull/2060#discussion_r1193500569 ?

Reply:
yes it's the same